### ISSUE TITLE
tests fail on FreeBSD

### ISSUE THREAD

**Issue by joshrabinowitz** [sobolevn/git-secret#244]
We should have some tests of git-secret on some flavor(s) of *BSD.

Can we do this with travis-ci?

**Comment by joshrabinowitz** [comment]
Steps to reproduce:
* Install FreeBSD (I used 11.2 RELEASE on VirtualBox)
* install gpg, gawk, bash, gmake from pkg repos ('make' doesn't work at all)
* fetch git-secret source from github 
* cd git-secret
* gmake clean build test

What happens?
-------------
* all tests fail

What were you expecting to happen?
----------------------------------
* tests pass

Any logs, error output, etc?
----------------------------


````
not ok 1 run 'add' normally
...
#  `install_fixture_key "$TEST_DEFAULT_USER" failed"
...
and so on. All tests fail.
````

What versions of software are you using?
----------------------------------------

**Operating system:** (`uname -a`): FreeBSD freebsd.home.examp.com 11.2-RELEASE FreeBSD 11-2-RELEASE #0 r335510: Fri Jun 22 04:09:26 UTC 2018  root@releng2.nyi.freebsd.org:/usr/obj/src/sys/GENERIC  i386 

**gpg**: 2.2.9
**gawk**: 4.1.4
**gmake**: 4.2.1
**bash**: 4.4.23

**Comment by joshrabinowitz** [comment]
See  #260

**Comment by sobolevn** [comment]
Merged #260
